extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/binder/grpc/ServerGrpcMetrics.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.micrometer.runtime.binder.grpc;
+
+import javax.inject.Singleton;
+
+import io.grpc.Metadata;
+import io.grpc.ServerCall;
+import io.grpc.ServerCallHandler;
+import io.grpc.ServerInterceptor;
+import io.micrometer.core.instrument.Metrics;
+import io.micrometer.core.instrument.binder.grpc.MetricCollectingServerInterceptor;
+import io.quarkus.grpc.GlobalInterceptor;
+
+@GlobalInterceptor
+@Singleton // gRPC uses final methods
+public class ServerGrpcMetrics extends MetricCollectingServerInterceptor implements ServerInterceptor {
+
+    public ServerGrpcMetrics() {
+        super(Metrics.globalRegistry);
+    }
+
+    @Override
+    public <Q, A> ServerCall.Listener<Q> interceptCall(ServerCall<Q, A> call, Metadata requestHeaders,
+            ServerCallHandler<Q, A> next) {
+        return super.interceptCall(call, requestHeaders, next);
```

Comment:
@michalszynkiewicz I've added this method to ease the debugging. It will be removed.

Reply:
@cescoffier for interceptors defined within an extension (i.e., in Quarkus), you need to register them with `AdditionalGlobalInterceptorBuildItem`.
An alternative would probably be to add them to index with `AdditionalIndexedClassesBuildItem` but I haven't tried it.

Reply:
here are examples: https://github.com/cescoffier/quarkus/blob/a528d006dc179238bbd61fa7d8877e54b5418a70/extensions/grpc/deployment/src/main/java/io/quarkus/grpc/deployment/GrpcServerProcessor.java#L365-L373

Reply:
Thanks!

Reply:
unfortunately, we don't have a dependency on the gRPC deployment module (on purpose)...

Reply:
so let's try with `AdditionalIndexedClassesBuildItem`.

Reply:
It works!